commit b8bce89be7fb5bffe5fef4a2782ca9b2b107eaac
Author: Jia Tan <jiat0218@gmail.com>
Date:   Sat Feb 4 12:01:23 2023 +0800

    xz: Add a comment clarifying the use of start_time in mytime.c.

diff --git a/src/xz/mytime.c b/src/xz/mytime.c
index 8d5e994f..917eff0f 100644
--- a/src/xz/mytime.c
+++ b/src/xz/mytime.c
@@ -20,6 +20,11 @@
 
 uint64_t opt_flush_timeout = 0;
 
+// The start_time variable will not represent the actual start time
+// if mytime_sigtstp_handler() executes. The signal handler measures
+// the amount of time spent stopped and adds it to start_time.
+// So, care must be taken in the future if the actual start time needs
+// to be displayed for any reason.
 #ifdef USE_SIGTSTP_HANDLER
 static volatile uint64_t start_time;
 #else